commands/game/rwhost.js

```diff
@@ -0,0 +1,23 @@
+const db = require("quick.db")
```

Comment:
delete this file or the other one. two comads with the same name don't work. one file will never be used. always the first one in the collection